cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -470,3 +470,13 @@ func (o *objectGraph) setCRSTenant(node, tenant *node) {
 		}
 	}
 }
+
+// warnVirtualNode logs warning if some node are still virtual
+func (o *objectGraph) checkVirtualNode() error {
+	for _, node := range o.uidToNode {
+		if node.virtual {
+			return errors.Errorf("node %v/%v is still virtual", node.identity.Kind, node.identity.Name)
```

Comment:
What about
```suggestion
			return errors.Errorf("node %v/%v wont be moved because not included in GVK considered for move", node.identity.Kind, node.identity.Name)
```